src/web/releases.rs

```diff
@@ -431,45 +375,14 @@ pub fn releases_failures_by_stars_handler(req: &mut Request) -> IronResult<Respo
 
 pub fn owner_handler(req: &mut Request) -> IronResult<Response> {
     let router = extension!(req, Router);
-    // page number of releases
-    let page_number: i64 = router
-        .find("page")
-        .and_then(|page_num| page_num.parse().ok())
-        .unwrap_or(1);
-    let owner_route_value = router.find("owner").unwrap();
-
-    let (owner_name, releases) = {
-        let mut conn = extension!(req, Pool).get()?;
-
-        // We need to keep the owner_route_value unchanged, as we may render paginated links in the page.
-        // Changing the owner_route_value directly will cause the link to change, for example: @foobar -> foobar.
-        let mut owner = owner_route_value;
-        if owner.starts_with('@') {
-            owner = &owner[1..];
-        }
-        get_releases_by_owner(&mut conn, page_number, RELEASES_IN_RELEASES, owner)
-    };
-
-    if releases.is_empty() {
-        return Err(Nope::OwnerNotFound.into());
+    let mut owner = router.find("owner").unwrap();
+    if owner.starts_with('@') {
+        owner = &owner[1..];
     }
-
-    // Show next and previous page buttons
-    let (show_next_page, show_previous_page) = (
-        releases.len() == RELEASES_IN_RELEASES as usize,
-        page_number != 1,
-    );
-
-    ViewReleases {
-        releases,
-        description: format!("Crates from {}", owner_name),
-        release_type: ReleaseType::Owner,
-        show_next_page,
-        show_previous_page,
-        page_number,
-        owner: Some(owner_route_value.into()),
+    match format!("https://crates.io/users/{}", owner).parse() {
```

Comment:
do you mind adding a small test for this redirect?

Reply:
I added a test for the happy path. I couldn't figure out a URL that would fail to parse based solely on the path component so I don't have a test for the error path, but if you can think of one let me know and I'll add it.